src/pages/academy/blog/posts/_style.js

```diff
@@ -449,21 +472,27 @@ export const MobileBreadcrumbsWrapper = styled.div`
 `
 export const StyledBreadcrumbsLink = styled(LocalizedLinkText)`
     font-weight: normal;
+    white-space: nowrap;
     font-size: 14px;
     line-height: 20px;
     color: var(--color-grey-5);
+    padding-right: 13px;
 `
 export const StyledBreadcrumbsTitle = styled(Text)`
+    display: flex;
+    white-space: nowrap;
     font-weight: normal;
     font-size: 14px;
     line-height: 20px;
     color: var(--color-black-3);
+    padding: 0 0 0 13px;
```

Comment:
```suggestion
    padding-left: 13px;
```
For consistency as component above.